operator/v1/types.go

```diff
@@ -52,6 +52,7 @@ type OperatorSpec struct {
 	// logLevel is an intent based logging for an overall component.  It does not give fine grained control, but it is a
 	// simple way to manage coarse grained logging choices that operators have to interpret for their operands.
 	// +optional
+	// +kubebuilder:validation:Enum=Normal;Debug;Trace;TraceAll
```

Comment:
it makes sense to set the enum on the type itself right? here https://github.com/openshift/api/blob/master/operator/v1/types.go#L80

so that LogLevel and OperatorLogLevel both get the enum in docs...

Reply:
I'm not sure how to do that, can you provide me an example with your suggestion?

Reply:
Makes sense, fixed!